browser/src/control/Control.Tabs.js

```diff
@@ -324,6 +324,9 @@ L.Control.Tabs = L.Control.extend({
 		if (part !== this._map._docLayer._selectedPart) {
 			this._setPartIndex(part);
 		}
+
+		if (this._map._docLayer.isAutoFillPopupOpen)
```

Comment:
please just fire an event and do the "if ( ... ) " in the handler

Reply:
this is removed in the new commits: no need x 2